Gruntfile.js

```diff
@@ -292,10 +292,9 @@ module.exports = function(grunt) {
     },
 
     shell: {
-      'npm-install': {
-        command: 'node scripts/npm/check-node-modules.js'
+      'install-node-dependencies': {
+        command: 'yarn'
```

Comment:
I did try of course. I didn't take your word for it 😛
It "stabilizes" at 2secs on my machine, which is OK (for me at least).